activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Disallow raw SQL in `group` and `calculate` methods
```

Comment:
```suggestion
*   Deprecate raw SQL in `group` and `calculate` methods without marking the SQL as safe.
```

Given we already removed the deprecation for this in Rails 6.1, we can't disallow without deprecating, so we need to change the code to show a deprecation instead of raising an error.

Also, let's expand with examples of how to make it work again.